Evaluate-ModelarDB-Compression/main.py

```diff
@@ -0,0 +1,281 @@
+""" Script for evaluating ModelarDB with different data sets. """
+
+
+import sys
+import math
+import time
+import signal
+import tempfile
+import subprocess
+
+import numpy
+from pyarrow import parquet
+from pyarrow import flight
+
```

Comment:
Maybe run a formatting tool on the file. I think there should only be a single newline between imports and module constants.

Reply:
It seems that `Evaluate-ModelarDB-Changes/main.py` might have the same formatting issues so maybe also fix it there.

Reply:
The file was already formatted using [black](https://github.com/psf/black), so I manually removed one of the empty lines from `Evaluate-ModelarDB-Compression/main.py` and `Evaluate-ModelarDB-Changes/main.py`.